### PR TITLE
added Docker building instructions

### LINKED ISSUE #41: [README.md] Adding Docker instructions for local building
Jekyll requires different stuff in order to be used (e.g. Ruby and Gems).  
For those (like me :D) who don't like to install the whole Jekyll package and its dependencies, a Docker one-shot ephemeral container works just fine for local building.

To build using `jekyll/jekyll` Docker image, use this command:
```sh
docker run --rm -it \
-p 4000:4000 \
--volume "/absolute/path/to/site/repo":/srv/jekyll \
jekyll/jekyll jekyll serve
```

Adding this method in the README.md would be great.

### PR DESCRIPTION
**Pull request by ar3s3ru**
Closes #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +43,36 @@ To get starting developing, we really recommend to clone the website locally and
...
+2. Clone this repository (you need `git` on your machine)
+```
+git clone https://github.com/gdgpisa/gdgpisa.github.io.git
+cd gdgpisa.github.io/
+```
+
+3. Run a Docker container with `jekyll/jekyll` image and serve the site
+```
+docker run --rm -it \
+-p 4000:4000 -v "$PWD":/srv/jekyll
```
Comment (cortinico) [inline README.md]: `\` is missing at the end of this line.

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 * [GDG Pisa's public website](#gdg-pisas-public-website-)
   + [Feature](#feature-)
   + [Getting Started (builing the website locally)](#getting-started-builing-the-website-locally-)
+     + [Docker](#docker)
```
Comment (cortinico) [inline README.md]: Please move `Docker` as last entry in this list.

**Comment by ar3s3ru** [comment]
@cortinico fixed

**Comment by cortinico** [comment]
Tested locally :) All works smoothly

**Pull request merged**